backend/mora/service/__init__.py

```diff
@@ -7,15 +7,20 @@
 #
 
 from . import address
+from . import association
```

Comment:
I assume this will generate a lot of warnings by `flake8`? We should probably have a `nocov` or `noqa` comment, or perhaps include them in `__all__` if that silences it.